plugins/trivy/pkg/models/image.go

```diff
@@ -40,6 +41,12 @@ type TrivyResults struct {
 type TrivyMetadata struct {
 	ImageID     string
 	RepoDigests []string
+	ImageConfig TrivyImageConfig
```

Comment:
Also add this in schema for testing.

Reply:
where is it exactly?

Reply:
I have updated the `json.schema` [here](https://github.com/FairwindsOps/insights-plugins/pull/654/files#diff-5dfdabd8025959e338b8c2e67dcd3ac01caba84141fd1f5b7d2f26aca357fade), is that what you meant?

Reply:
Because the struct `TrivyMetadata` is actually modeling Trivy's command line JSON output that we ingest, not the trivy report output